recipe/meta.yaml

```diff
@@ -58,13 +60,13 @@ test:
     {% endfor %}
 
     # CLI tests
-    - genbrk --help  # [not win]
-    - gencfu --help  # [not win]
-    - gencnval --help  # [not win]
-    - gendict --help  # [not win]
-    - icuinfo --help  # [not win]
+    - genbrk --help
+    - gencfu --help
+    - gencnval --help
+    - gendict --help
+    - icuinfo --help
     - icu-config --help  # [not win]
```

Comment:
No there didn't seem to be.  I'm away from my computer so can't easily double check at the moment.